lib/teiserver/game/servers/balancer_server.ex

```diff
@@ -134,17 +133,31 @@ defmodule Teiserver.Game.BalancerServer do
     players = Battle.list_lobby_players(state.lobby_id)
```

Comment:
I'm sort of stuck here, I don't know how to mock this to return a fake list of players, so I'm ending up trying to spin up a real lobby with real players in order to try to get the BalancerServer to actually operate.

Not trying to test this exhaustively, just trying to smoke-test it.